src/test/java/com/icthh/xm/tmf/ms/communication/config/TenantConfigMockConfiguration.java

```diff
@@ -42,4 +43,9 @@ public TenantConfigRepository tenantConfigRepository() {
     public BusinessTimeConfigService businessTimeConfigService() {
         return mock(BusinessTimeConfigService.class);
     }
+
+    @Bean("TTLRuleConfigService")
```

Comment:
why do we need bean name definition here?

Reply:
yes, spring manages this bean as "TTLRuleConfigService" and cannot find the factory method without annotation. Just try to delete it and run tests